tests/test_logging.py

```diff
@@ -76,17 +83,19 @@ def set_user_config_env(temp_user_logging_config):
     original_config_env = os.getenv("OSMOSE_USER_CONFIG", None)
     os.environ["OSMOSE_USER_CONFIG"] = str(temp_user_logging_config.parent)
     importlib.reload(osekit)
+    setup_logging()
     yield
     if original_config_env:
         os.environ["OSMOSE_USER_CONFIG"] = original_config_env
     else:
         del os.environ["OSMOSE_USER_CONFIG"]
     importlib.reload(logging)
     importlib.reload(osekit)
```

Comment:
I'm wondering if these lines are still necessary (at least the two `importlib.reload(osekit)` since we can now call `setup_logging()` on-demand.